src/transformers/pipelines/video_classification.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Union
+
+from ..utils import add_end_docstrings, is_decord_available, is_torch_available, logging, requires_backends
+from .base import PIPELINE_INIT_ARGS, Pipeline
+
+
+if is_decord_available():
+    import numpy as np
+
+    from decord import VideoReader, cpu
+
+
+if is_torch_available():
+    from ..models.auto.modeling_auto import MODEL_FOR_VIDEO_CLASSIFICATION_MAPPING
+
+logger = logging.get_logger(__name__)
+
+
+@add_end_docstrings(PIPELINE_INIT_ARGS)
+class VideoClassificationPipeline(Pipeline):
+    """
+    Video classification pipeline using any `AutoModelForVideoClassification`. This pipeline predicts the class of a
+    video.
+
+    This video classification pipeline can currently be loaded from [`pipeline`] using the following task identifier:
+    `"video-classification"`.
+
+    See the list of available models on
+    [huggingface.co/models](https://huggingface.co/models?filter=video-classification).
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        requires_backends(self, "vision")
+        self.check_model_type(MODEL_FOR_VIDEO_CLASSIFICATION_MAPPING)
+
+        self.frame_sample_rate = kwargs.pop("frame_sample_rate", 4)
```

Comment:
Need this information but its not in model config or preprocessor. Whats best place for this to be?